content/kit/webcam.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Webcam
+---
+
+To see the outside world, a webcam is used to capture images for them to be processed. The webcam in the kit is a [TeckNet C016](http://www.tecknet.co.uk/c016.html).
+
+![The Webcam](http://www.tecknet.co.uk/media/catalog/product/cache/1/thumbnail/9df78eab33525d08d6e5fb8d27136e95/c/0/c016_webcam_4_.jpg?width=30pc)
+
+## Focus
+The webcam has a manual focus ring around the lens. During standard operation, this lens may be knocked, and move the image out of focus, which can seriously reduce the accuracy of the [vision](/api/vision/). 
+
+It's recommended to manually focus the camera using a separate computer, and then use a __non permanent__ method (a small drop of hot glue is great for this) to prevent it moving.
+
+{{% notice warning %}}
+You will likely need to manually focus the webcam before you first use it
```

Comment:
Can we put at least a full stop here? xD The lack of punctuation at the end of the sentence is bugging me :P #Pedantic

Reply:
Yeah, I meant that the exclamation mark should be replaced with a period, not just removed completely.